src/ntfs/write.rs

```diff
@@ -179,6 +177,229 @@ pub fn write_fares_v1(
     Ok(())
 }
 
+fn write_fares_v1_from_v2(
+    base_path: &path::Path,
+    tickets: &CollectionWithId<Ticket>,
+    ticket_uses: &CollectionWithId<TicketUse>,
+    ticket_prices: &Collection<TicketPrice>,
+    ticket_use_perimeters: &Collection<TicketUsePerimeter>,
+    ticket_use_restrictions: &Collection<TicketUseRestriction>,
+) -> Result<()> {
+    fn has_no_transfer(ticket_uses: &CollectionWithId<TicketUse>, ticket_use_id: &String) -> bool {
+        if let Some(ticket_use) = ticket_uses.get(ticket_use_id) {
+            if (ticket_use.max_transfers.is_none() || ticket_use.max_transfers == Some(0))
+                && ticket_use.boarding_time_limit.is_none()
```

Comment:
I'm not sure `boarding_time_limit` and `alighting_time_limit` are relative to a transfer. I thought you can have a `boarding_time_limit` on the Ticket Use, even if no transfer is allowed.